src/styles/index.scss

```diff
@@ -1,3 +1,133 @@
-body {
+$main-color: #060b35;
+$second-color: #616070;
+$blue-color: #00acdc;
+$bacgground-color: #fff;
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: $second-color;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 406px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $bacgground-color;
+
+  &__image {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__header {
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__code {
+    margin: 4px 0 0;
+  }
+
+  &__info {
+    margin-top: 24px;
+    display: flex;
+    font-display: row;
+    justify-content: space-between;
+  }
+
+  &__content {
+    font-size: 12px;
+    font-style: normal;
+    line-height: 18px;
+    color: $second-color;
+  }
+
+  &__raiting {
+    margin-top: 16px;
+    display: flex;
+    font-display: row;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    text-align: right;
+    font-style: normal;
```

Comment:
`normal` it's the default value. No need to write it
```suggestion
```

Reply:
Despite of html properties?
html {
  font-family: "Roboto", Helvetica, sans-serif;
  font-weight: 400;
  font-size: 10px;
  line-height: 14px;
  color: $second-color;
}